lib/static/components/modals/screenshot-accepter/header.jsx

```diff
@@ -96,6 +99,11 @@ export default class ScreenshotAccepterHeader extends Component {
         event.preventDefault();
 
         const {images, retryIndex, onScreenshotAccept} = this.props;
+
+        if (retryIndex === null) {
```

Comment:
In case if all images are accepted. Yes, button "accept" will be disabled, but user still could call this method by pressing the spacebar button

https://github.com/gemini-testing/html-reporter/blob/master/lib/static/components/modals/screenshot-accepter/index.jsx#L99-L105